docs/guides/es2k/build_p4_artifacts.md

```diff
@@ -0,0 +1,92 @@
+# Guide to build P4 artifacts
+
+- [1. Overview](#1-overview)
+- [2. Install p4c, cpt and p4-sde RPMs](#2-install-p4c-cpt-and-p4-sde-rpms)
+- [3. Generate P4 artifacts](#3-generate-p4-artifacts)
+- [4. Generate forwarding pipeline binary](#5-generate-forwarding-pipeline-binary)
+   - [4.1 Prepare configuration files](#4-1-prepare-configuration-files)
+   - [4.2 Generate binary](#4-2-generate-binary)
+
+## 1. Overview
+This document explains how to install Intel® IPU E2100 P4 Compiler `p4c-pna-xxp` for Intel® Configurable Pipeline Tool `cpt` to build P4 artifacts. It also explains how to use `tdi_pipeline_builder` utility to generate a binary to program forwarding pipeline config using P4Runtime client `p4rt-ctl`
```

Comment:
should there be an "and" after `p4c-pna-xxp` or it is "for" only??

Reply:
Good catch, its `and`